client/signup/is-flow.ts

```diff
@@ -0,0 +1,21 @@
+import config from '@automattic/calypso-config';
+
+export const isReskinnedFlow = ( flowName: string ) => {
+	return config< string[] >( 'reskinned_flows' ).includes( flowName );
+};
+
+export const isP2Flow = ( flowName: string ) => {
+	return flowName === 'p2' || flowName === 'p2v1';
+};
+
+export const isVideoPressFlow = ( flowName: string ) => {
```

Comment:
According to the existing types, `flowName` can also be a `null`. Should we adjust either side?

Reply:
Just pushed a fix for that, having tracked down that `flow` is never `null`.

Reply:
Perfect, thanks!